share/dictionary.alcatel.sr

```diff
@@ -371,10 +371,10 @@ ATTRIBUTE	Alc-SLAAC-IPv6-Pool			181	string
 # DHCP6 attributes
 ATTRIBUTE	Alc-ToServer-Dhcp6-Options		191	octets
 ATTRIBUTE	Alc-ToClient-Dhcp6-Options		192	octets
-ATTRIBUTE	Alc-v6-Preferred-Lifetime		200	integer
-ATTRIBUTE	Alc-v6-Valid-Lifetime			201	integer
-ATTRIBUTE	Alc-Dhcp6-Renew-Time			202	integer
-ATTRIBUTE	Alc-Dhcp6-Rebind-Time			203	integer
+ATTRIBUTE	Alc-v6-Preferred-Lifetime		200	date
```

Comment:
https://infocenter.nokia.com/public/7750SR150R5A/index.jsp?topic=%2Fcom.sr.radius%2Fhtml%2Fsros_radius_attrib.html

these all appear to be time deltas :(

Reply:
But, we don't have time_delta in v3.

Reply:
Ah! nevermind.